### PR TITLE
[luci] Support quantization of floor

### PR DESCRIPTION
**Pull request by dayo09**
This commit supports quantization of floor operation
and the unit test.

ONE-DCO-1.0-Signed-off-by: Dayoung Lee <dayg502@gmail.com>

-------------

For #6367

### CODE REVIEW

**Comment by dayo09** [comment]
Reference : https://www.tensorflow.org/api_docs/python/tf/math/floor

File: compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp [hunk 1]
```diff
@@ -941,6 +941,7 @@ void quantize_const_inputs(luci::CircleNode *node, loco::DataType output_type)
     case luci::CircleOpcode::SOFTMAX:
     case luci::CircleOpcode::SUB:
     case luci::CircleOpcode::TANH:
+    case luci::CircleOpcode::FLOOR:
```
Comment (jinevening) [inline compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp:944]: Can you replace this to follow alphabetical order?

Comment (jinevening) [reply compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp:944]: Except this, everything looks good to me.

**Review by jinevening — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**